### ISSUE TITLE
[JOSS] Installation Instructions

### ISSUE THREAD

**Issue by joezuntz** [jronayne/PyTransport#7]
The installation instructions in the User Guide are quite hard to follow - I found it very difficult!  Could you write something that makes it clearer with step-by-step instructions what needs to be done, maybe starting from one of the examples?

**Comment by jronayne** [comment]
I have added a quick install section in the `README.md` file of the main repository. It is hopefully a lot clearer than the previous guide in the user documentation. Thanks again for raising this issue.

**Comment by joezuntz** [comment]
Thanks - easier now.
